<?xml version="1.0" encoding="UTF-8" ?>

<launch>

  <!-- PHASE 2: Plane segmentation orientation correction-->
  <node pkg="dafan_docking" type="orientate_plane" name="orientate_plane" output="screen">
    <rosparam file="$(find dafan_docking)/config/phase2_orientate_plane.yaml"/>
    <remap from="camera/depth/color/points" to="/d435/depth/color/points"/>
  </node>

  <!-- Planner Adjuster PID Controller -->
  <!-- <node pkg="dafan_docking" type="dafan_planner_adjuster" name="dafan_planner_adjuster" output="screen">
    <rosparam file="$(find dafan_docking)/config/planner_adjuster.yaml" command="load" />
    <remap from="/cmd_vel" to="/cmd_vel_mux/autonomous"/>
  </node> -->

</launch>
